src/main/java/racingcar/service/RacingCarWebService.java

```diff
@@ -0,0 +1,36 @@
+package racingcar.service;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import racingcar.domain.RacingCar;
+import racingcar.dto.RacingResultResponse;
+import racingcar.dto.RacingStartDto;
+import racingcar.jdbctemplate.InsertDao;
+import racingcar.jdbctemplate.QueryDao;
+
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+public class RacingCarWebService {
+    private final InsertDao insertDao;
+    private final QueryDao queryDao;
+    private final RacingCarService racingCarService;
+
+    public RacingResultResponse playRacingWebGame(RacingStartDto racingStartDto) {
+
+        RacingCar racingCar = racingCarService.playRacingGame(racingStartDto, queryDao.getNextRound());
+
+        insertDao.insertRacingHistory(racingCar, racingStartDto.getTrial());
+        insertDao.insertWinnerHistory(racingCar);
+
+        return new RacingResultResponse(
+                racingCar.getWinner().getCarNames(),
+                racingCar.getCars()
```

Comment:
디미터 법칙을 지켜서 우승자의 이름을 가져오면 어떨까요?
```suggestion
                racingCar.getWinnerNames(),
                racingCar.getCars()
```